libs/wingsdk/src/shared/bundling.ts

```diff
@@ -72,13 +72,13 @@ export function createBundle(
     new TextDecoder().decode(esbuild.outputFiles[0].contents)
   );
   if (sourcemapData.sourceRoot) {
-    sourcemapData.sourceRoot = normalPath(sourcemapData.sourceRoot);
+    sourcemapData.sourceRoot = normalPath(resolve(sourcemapData.sourceRoot));
   }
 
   for (const [idx, source] of Object.entries(
     sourcemapData.sources as string[]
   )) {
-    sourcemapData.sources[idx] = normalPath(source);
+    sourcemapData.sources[idx] = normalPath(resolve(source));
   }
```

Comment:
Without this change, source paths were getting messed up on my system, causing the files to not be found - I was getting stack traces like this:

```
fail ┌ inflight_stacktraces.test.wsim » root/env0/test:assert
     │ child process.argv: [
     │   '/Users/chrisr/.volta/tools/image/node/20.11.0/bin/node',
     │   '/var/folders/q3/0yxxhjr95vx4yfkk3lvy6dj40000gn/T/wing-bundles-gefQdo/index.js'
     │ ]
     │ Error: assertion failed: false
     │ at /private/Users/chrisr/dev/wing1/examples/tests/error/inflight_stacktraces.test.w:7:3
     │ at handler /private/Users/chrisr/dev/wing1/examples/tests/error/target/test/inflight_stacktraces.test.wsim/.wing/handler_c8b175d2.sandbox.js:19:25
     └ at /private/Users/chrisr/dev/wing1/examples/tests/error/target/test/inflight_stacktraces.test.wsim/.wing/handler_c8b175d2.sandbox.js:24:19
```

I might be doing something wrong though

Reply:
Hmm, this only fixes it when I run via the compiled Wing CLI and not when it's run by the hangar.